StoreKitUnitTests/TrialOrIntroPriceEligibilityCheckerSK1Tests.swift

```diff
@@ -75,6 +75,35 @@ class TrialOrIntroPriceEligibilityCheckerSK1Tests: StoreKitConfigTestCase {
         expect(receivedEligibilities.count) == 1
     }
 
+    func testSK1EligibilityMapAllProductsWithIntroEligibility() throws {
```

Comment:
this test would fail right now in `main`, right?

Reply:
Yup yup! I wrote this test and commented out my fixed and it failed. Reapplied my fix and then it passed 😊